clients/wallet/src/horizon/horizon.rs

```diff
@@ -289,6 +289,7 @@ pub async fn listen_for_new_transactions<T, U, Filter>(
 	issue_map: Arc<RwLock<T>>,
 	memos_to_issue_ids: Arc<RwLock<U>>,
 	filter: Filter,
+	#[cfg(any(test, feature = "testing-utils"))] last_cursor: PagingToken,
```

Comment:
this is to address the "could not find event" errors. 
The `testing-utils` adds an extra param for a function that can define the "last known cursor".

See its usage in clients/vault/tests/vault_integration_tests.rs